src/main/rascal/refactor/TextEdits.rsc

```diff
@@ -31,13 +31,8 @@ extend analysis::diff::edits::TextEdits;
 
 import util::Maybe;
 
-alias ChangeAnnotationId = str;
-
 data ChangeAnnotation
```

Comment:
I'm not so sure we should have this as part of this framework, smells more like a thing we either have in VS Code, or try to bring to `TextEdits` module.

Reply:
Removed